src/components/TaskList.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Task from "./Task";
+import TaskComposer from "./TaskComposer";
+
+export default class TaskList extends React.Component {
+  render() {
+    const {
+      tasks,
+      updateTask,
+      addTaskToDo,
+      deleteTask,
+      resetLocalStorage,
+      moveTaskOpen,
+    } = this.props;
+
+    return (
+      <div className="task-list content">
+        <h1>
+          <strong>Open Tasks</strong>
+        </h1>
+        <h2 className="header-2">Number of tasks: {tasks.length}</h2>
+        <div>
+          <button onClick={resetLocalStorage}>Reset to Default</button>
+        </div>
+        <TaskComposer addTask={addTaskToDo} taskLength={tasks.length} />
+        {tasks && tasks.length > 0 ? (
```

Comment:
Since tasks is an array, the first condition is always true. `[]` is a truthy value in JavaScript

Reply:
```suggestion
        {tasks.length ? (
```

this would be sufficient here